site/en/guides/build/logs.md

```diff
@@ -220,14 +218,14 @@ For example, to enable this for an nrf52840 connected to a Linux host:
 
 1.  Edit the configuration file for the platform and change the log output to
     NCP Spinel. For nrf52840, this is
-    `/examples/platforms/nrf528xx/nrf52840/openthread-core-nrf52840-config.h`:
+    `/src/nrf52840/openthread-core-nrf52840-config.h`:
 
         #define OPENTHREAD_CONFIG_LOG_OUTPUT OPENTHREAD_CONFIG_LOG_OUTPUT_NCP_SPINEL
```

Comment:
What configuration is suggested as of today? `OPENTHREAD_CONFIG_LOG_OUTPUT_NCP_SPINEL` is not available any more.

Reply:
> #error "OPENTHREAD_CONFIG_LOG_OUTPUT_NCP_SPINEL is removed, use OPENTHREAD_CONFIG_LOG_OUTPUT_APP instead"

have you tried OPENTHREAD_CONFIG_LOG_OUTPUT_APP?

Reply:
I've tried it but I still didn't see any log output in spinel cli.